test/integration/index.test.js

```diff
@@ -23,7 +23,10 @@ const mockDefaults = {
     logger,
 };
 
-const createMockServer = (opts) => supertest(App({ ...mockDefaults, ...opts }).callback());
+const createMockServer = async (opts) => {
+    const app = await App({ ...mockDefaults, ...opts });
```

Comment:
It looks like `App` is not async? Also, this looks like quite unconventional capitalisation; `App` looks and is named like a class constructor but is really a function. Of course this is a pretty arbitrary distinction in js, but I think it's confusing to the reader.

Reply:
I don't know where I got the idea that this wasn't async. Sorry.